server/src/email/email-util.es6

```diff
@@ -47,32 +47,35 @@ emailUtil.transporter = nodemailer.createTransport(smtpConfig);
  * @param {boolean} attachments - email contains attachments
  */
 emailUtil.send = (to, subject, body, html = false, attachments = false) => {
-  const bodyAndNoReply = `${body}
+  return new Promise((resolve) => {
```

Comment:
Ha, nice, I was wondering if this was changed to match the expectations in `christmasTree.generateRulesAndEmail`.  :high-five: